src/aap_eda/services/ruleset/activate_rulesets.py

```diff
@@ -160,7 +160,6 @@ def activate(self, activation: models.Activation) -> None:
             else:
                 raise ActivationException(f"Unsupported {deployment_type}")
 
-            instance.refresh_from_db()
             if str(instance.status) == ActivationStatus.COMPLETED.value:
```

Comment:
@hsong-rh Wouldn't this be a stale instance unless we do a refresh and fetch the updated object from the DB?

Reply:
@mkanoor It doesn't matter whether the instance is stale or not, if it's deleted already